### ISSUE TITLE
Incorrect validation error on !uppercase/!lowercase functions usage

### ISSUE THREAD

**Issue by pgatilov** [mulesoft/api-workbench#103]
In this example:

```
#%RAML 1.0
title: Test
version: 1
baseUri: /api/{version}
traits:
  notAlwaysPresent:
    responses:
      404:
        description:  <<param | !uppercase>> cannot be found
/file:
  description: Represents a collection of files
  /{fileId}:
    description: Represents an individual file
    is:   { notAlwaysPresent: {param : test} }
    get:
      description: Gets a file
      responses:
        200:
          description: The request completed successfully.
          body:
            application/octet-stream:
              type: file
```
I get an error saying 'unknown function applied to parameter at line 9 col 9'. The same error appears for all functions in uppercase/lowercase group.

However, it does work with singularize/pluralize. Also, uppercase seems to work just fine if I open API Console for the file inside Atom.

Atom: 1.4.1
Api-Workbench: 0.8.14

**Comment by sichvoge** [comment]
Hi @pgatilov, thanks for raising that issue.

I can confirm that behaviour. The workbench complains about all functions mentioned [here](http://docs.raml.org/specs/1.0/#resource-type-and-trait-parameters) except `! singularize` and `!pluralize`. Although there is no error using the standalone parser. We will investigate!

**Comment by ddenisenko** [comment]
Fixed. Fix is published in 0.8.15.

**Comment by pgatilov** [comment]
Just pulled the new package. I can confirm the issue has been fixed. Thanks.
